deploy/src/erc_to_erc/preDeploy.js

```diff
@@ -0,0 +1,18 @@
+const { web3Foreign } = require('../web3')
+const { ERC20_TOKEN_ADDRESS, ERC20_EXTENDED_BY_ERC677 } = require('../loadEnv')
+const { abi } = require('../../../build/contracts/ERC677BridgeToken.json')
+
+async function preDeploy() {
```

Comment:
Is it possible to have this check in `validations` of the `src/loadEnv.js` file?

Reply:
It is possible, but it will increase complexity on the usage of `loadEnv.js`. Right now this file reads the env variables and perform the validations synchronously. If we perform requests as part of these validations, the file will export a promise that will return the env variables instead of just returning the env variables. So on every file it is used we will have to handle this and on some utils files such us `src/web3.js` the exported values will also become a Promise because it need the env variables to generate and export objects as `web3Home` and `web3Foreign`. I think it would be a good idea to keep `loadEnv.js` as it is. If you want to avoid adding this new `preDeploy` file, maybe we can perform the check inside the foreign deploy and deploy first foreign instead of home on this bridge mode

Reply:
Thanks for explanation. Let's leave `preDeploy` as you implemented it already.